### PR TITLE
fix the reading of the "entries-read" field in XInfoStreamFull

### PR DESCRIPTION
**Pull request by monkey92t**
#2592

### CODE REVIEW

File: command.go [hunk 1]
```diff
@@ -2373,7 +2373,7 @@ func readStreamGroups(rd *proto.Reader) ([]XInfoStreamGroup, error) {
 				}
 			case "entries-read":
 				group.EntriesRead, err = rd.ReadInt()
-				if err != nil {
+				if err != nil && err != Nil {
```
Comment (SoulPancake) [inline command.go:2376]: @monkey92t Shouldn't this be redis.Nil?

Comment (monkey92t) [reply command.go:2376]: > @monkey92t Shouldn't this be redis.Nil?

??????  Your question is quite strange...

We are modifying the redis package itself, so there is no need to use `redis.Nil`, we can directly use `Nil`.
For example, in the io package, we don't need to write `io.Reader`.

Comment (SoulPancake) [reply command.go:2376]: Thank you for your explanation, @monkey92t. I understand now, we are indeed within the context of the Redis package itself, so it does make sense to use Nil directly. I appreciate your patience and for correcting my misunderstanding. I'm still learning and your clarification really helps. 
I was confused as I had previously seen redis.Nil in the tests and now I realise that they are in a different package.

Comment (monkey92t) [reply command.go:2376]: @SoulPancake  Our test code belongs to the `redis_test` package. If you look at `commands_test.go`, you should be able to see it. Therefore, in the test files, we need to write `redis.Nil`.

**Review by SoulPancake — APPROVED** [review]

**Review by vmihailenco — APPROVED** [review]

**Pull request merged**